pandas/core/reshape/reshape.py

```diff
@@ -582,7 +587,7 @@ def stack(frame: DataFrame, level=-1, dropna: bool = True, sort: bool = True):
     stacked : Series or DataFrame
     """
 
-    def factorize(index):
+    def stack_factorize(index):
```

Comment:
linter didn't like that this was named `factorize` now that we import `factorize` as well, and rightfully so.